js/models/conversations.js

```diff
@@ -1061,7 +1068,19 @@
     makeMessagesLookup(messages) {
       return messages.reduce((acc, message) => {
         const { source, sent_at: sentAt } = message.attributes;
-        const key = this.makeKey(source, sentAt);
+
+        // Checking for notification messages without a sender
+        if (!source && message.isIncoming()) {
```

Comment:
What does ‘notification message’ mean for an `incoming` message without source?

Reply:
A notification messages is a safety number change, a disappearing messages timer change, etc.